### DISCUSSION TITLE
What should be the Content-Type of a NEON file response from an API?

### DISCUSSION THREAD

**Discussion by matronator** [nette/neon#66] [Q&A] [answered]
I'm working on an API for internal use and have an endpoint that returns JSON, YAML or NEON files from a folder and was wondering, what value should I use for the `Content-Type` HTTP header if returning a NEON file.

Any "official" recommendation/suggestion?

**Comment by mabar** [comment, marked as answer]
It afaik does not have an official type, so best way to go should be `application/x-neon`

**Reply by matronator** [reply]
Great, thanks! :)
